eng/pipelines/aggregate-reports.yml

```diff
@@ -82,7 +94,7 @@ stages:
             options: '$(DefaultOptions) -T 2C -Dinclude-non-shipping-modules'
             mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
             javaHomeOption: 'JDKVersion'
-            jdkVersionOption: $(JavaBuildVersion)
+            jdkVersionOption: $(JavadocSafeJavaBuildVersion) # TODO (vcolin7): Change to ${{ parameters.JavaBuildVersion }} once Javadoc migration to Java 21 is complete.
```

Comment:
can the comment be changed to a github issue which contains the more verbose changes required?

Reply:
Sure thing! Here you go: https://github.com/Azure/azure-sdk-for-java/issues/38267